galaxy_ng/app/pipelines.py

```diff
@@ -0,0 +1,47 @@
+from django.conf import settings
+from django.contrib.auth.models import Group
+
+
+def user_role(response, details, user=None, *args, **kwargs):
```

Comment:
Could you add some comments here explaining that these pipelines are used for social auth?